config/database.yml

```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: myflix_development
+  pool: 5
+  username: Eric
```

Comment:
If your `config/database.yml` is public like now, I recommend 100% to hide this to an ENV variable. You can use ERb in yml here. `<% ENV["USER"] %>` should work if you are on a mac.